packages/core/core/package.json

```diff
@@ -34,29 +34,44 @@
     "_build": "npm run solo -- build",
     "ci": "npm run qa",
     "clean": "npm run reset",
-    "lint": "eslint process.js src/",
-    "qa": "npm-run-all lint _build",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint process.js src/",
+    "// lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
```

Comment:
I'm assuming that `lint:ts` is kept disabled for now because various linter errors will be fixed in later PRs.

Even so, if in `packages/core/core` you run `tslint -c tslint.json \"src/**/*.ts\" --fix` it will cleanup quite a few automatically:

```
$ tslint -c tslint.json "src/**/*.ts" --fix
Fixed 116 error(s) in src/config.ts
Fixed 9 error(s) in src/configDefaults.ts
Fixed 16 error(s) in src/engine.ts
Fixed 15 error(s) in src/events.ts
Fixed 21 error(s) in src/plugin.ts
Fixed 27 error(s) in src/plugins.ts
Fixed 12 error(s) in src/services_monitor.ts
```

It might be worth including those fixes in this PR, but I'll leave it up to you, this isn't a change request.

Reply:
Thanks, adding this!

Reply:
Actually, it doesn't fix a lot of them, so I'll keep it disabled for now. Will get those changes in in another PR that addresses type satefy as well

Reply:
In fact, running this command doesn't give me the same output as yours

Reply:
Okay... but it should give identical output. I realize now the command example may have been unclear; more precisely:

```
$ cd packages/core/core
$ npx tslint -c tslint.json "src/**/*.ts" --fix
Fixed 116 error(s) in src/config.ts
Fixed 9 error(s) in src/configDefaults.ts
Fixed 16 error(s) in src/engine.ts
Fixed 15 error(s) in src/events.ts
Fixed 21 error(s) in src/plugin.ts
Fixed 27 error(s) in src/plugins.ts
Fixed 12 error(s) in src/services_monitor.ts
```

The output is what I got just now after re-fetching this PR branch.

Reply:
Thanks @michaelsbradleyjr for some reason, this didn't work before (even with `npx`), but now it did. Updating..